using-mobile-extensions/mobile-core/identity/identity-api-reference.md

```diff
@@ -205,6 +206,7 @@ If the provided URL is nil or empty, it is returned as is. Otherwise, the follow
 * The adobe\_mc attribute is an URL encoded list containing:
   * Experience Cloud ID \(ECID\)
   * Experience Cloud Org ID
+  * Analytics Tracking ID, if available from [Analytics](../../adobe-analytics/)
```

Comment:
Analytics Tracking ID (AID), if available from the [Analytics extension](..)

Reply:
Below (L211): The optional `adobe_aa_vid` attribute is the URL-encoded Analytics Custom Visitor ID (VID), if previously set in the Analytics extension.

Reply:
Do you think we should give an output example here?